apps/managedidentity/example_test.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package managedidentity_test
+
+import (
+	"context"
+	"fmt"
+
+	mi "github.com/AzureAD/microsoft-authentication-library-for-go/apps/managedidentity"
+)
+
+var miSystemassignedClient mi.Client
+var err error
+
+func ExampleNew() {
+	// System assigned Client
+	miSystemassignedClient, err = mi.New(mi.SystemAssigned())
+	if err != nil {
+		// TODO: Handle error
+	}
+	_ = miSystemassignedClient
```

Comment:
Is this needed to stop some static analyzer from spewing a warning ? Can we disable the analyzer instead for the entire file?

Reply:
Yea, or else the compiler wont compile as the variable is not used.